README.md

```diff
@@ -1,54 +1,201 @@
 # kubernetes-nmstate
-Host-networking for kubernetes driven by kubernetes and executed by nmstate.
+
+Host-networking configuration driven by Kubernetes and executed by
```

Comment:
There is inconsistency of usage between `host` and `node`. It will be good to try and be consistent with the names and perhaps add a note that they mean the same (almost :) )

Reply:
Renamed all hosts to nodes